requirements/base.txt

```diff
@@ -154,7 +156,7 @@ django-sendfile2==0.7.0
     # via django-privates
 django-sessionprofile==3.0.0
     # via open-api-framework
-django-setup-configuration==0.1.0
+django-setup-configuration==0.4.0
```

Comment:
Did you update these with `bin/compile_dependencies.sh -P django-setup-configuration`? I think we need to pin it in `base.in` for now, because `open-api-framework` doesn't pin it to `>=0.4.0` yet

Can you add `django-setup-configuration>=0.4.0` to `base.in` and recompile?